### DISCUSSION TITLE
How to make a simple timer

### DISCUSSION THREAD

**Discussion by solsticedhiver** [jcorporation/myMPD#1226] [Q&A]
I wanted to replace a simple bash script like:
```
sleep xx && mpc pause
```

I am seeing on github issues that there has been something done to implement timers in mympd.

But I can't find how to do it in the web UI. And not much elp in the doc.

**Comment by jcorporation** [comment]
Timers can be created easily in the web ui. Main menu -> Timers. If there is no entry, you must enable it in settings.

**Comment by solsticedhiver** [comment]
I want to stop the playback after a specific amount of time like 30 minutes.
All I can see is to make a timer with a specific time at said hour and minutes.

**Reply by jcorporation** [reply]
Create a second timer that stops playback or create a timer that calls script that creates an one-shot timer to stop playback.

**Reply by solsticedhiver** [reply]
OK. I'll try later today or tommorrow
